octobot_trading/modes/abstract_trading_mode.py

```diff
@@ -286,6 +476,18 @@ def get_required_candles_count(cls, tentacles_setup_config: tm_configuration.Ten
             common_constants.DEFAULT_IGNORED_VALUE
         )
 
+    @classmethod
+    async def get_backtesting_plot(cls, exchange, symbol, backtesting_id, optimizer_id,
+                                   optimization_campaign, backtesting_analysis_settings):
+        try:
+            import tentacles.Meta.Keywords.scripting_library as scripting_library
+            ctx = context_management.Context.minimal(cls, logging.get_logger(cls.get_name()), exchange, symbol,
+                                                     backtesting_id, optimizer_id,
+                                                     optimization_campaign, backtesting_analysis_settings)
+            return await scripting_library.default_backtesting_analysis_script(ctx)
+        except ImportError:
+            raise ImportError("scripting_library keywords are required")
```

Comment:
I don't understand why a TradingMode should have a plot method

Reply:
you are right, i'm moving it